config/routes.rb

```diff
@@ -3,7 +3,7 @@
   devise_for :users
 
   root controller: :images, action: :index
-
+  resources :checkout 
```

Comment:
Ok so we use this `resources` helper when we are talking about MODELS. That why we have one for `shopping_carts` and `shopping_cart_items`. However, we are *not* going to create `checkouts`. We should instead have a `checkout` method in the `ShoppingCartController` and create that route manually, nested under `shopping_carts`